ui/editor/src/ctrl.ts

```diff
@@ -47,7 +47,7 @@ export default class EditorCtrl {
 
     window.Mousetrap.bind('f', () => {
       if (this.chessground) this.chessground.toggleOrientation();
-      redraw();
+      this.onChange()
```

Comment:
Changing the `redraw()` to `onChange()` instead since we'd like to update the current url. `onChange()` already contains a call to `redraw()`.